src/components/Registration/Registration.jsx

```diff
@@ -0,0 +1,138 @@
+/* eslint-disable react/prop-types */
+import React, { useState } from 'react';
+import Button from '../../common/Button/Button';
+import Input from '../../common/Input/Input';
+
+// import styles
+import './Registration.scss';
+import { Link, useNavigate } from 'react-router-dom';
+
+const forbiddenSymbols = /[#$%^&]/;
+const validEmail =
+	/^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/;
```

Comment:
This constants and functions below should exist on your helpers folder as a suggestion. Using abstraction here will give you extra readability to the component. Same for Login.jsx file.